### PR TITLE
Added support for inverse barcodes

### PR DESCRIPTION
**Pull request by zalesak**

### CODE REVIEW

File: src/mlkit/barcodescanning/index.d.ts [hunk 1]
```diff
@@ -7,11 +7,11 @@ export interface MLKitScanBarcodesResultBounds {
   origin: {
     x: number;
     y: number;
-  },
+  };
```
Comment (EddyVerbruggen) [inline src/mlkit/barcodescanning/index.d.ts]: Ha, good catch!

**Comment by EddyVerbruggen** [comment]
Really impressive stuff!

Do you perhaps have an example of an inverse barcode you tested your implementation with? I'd love to give it a spin.

**Comment by zalesak** [comment]
Hi Eddy, sorry for late response.
Unfortunately, I have just specimen from drugs packages, from customers. I can`t publish it wide. But in principle, solution just inverse picture before sending it to ML KIT so it will work with every inverse barcode. 
We are using it on 2D matrix and QR Codes.

**Comment by EddyVerbruggen** [comment]
OK, thanks for the reply. Then I'm just going to assume it works ;)

**Comment by zalesak** [comment]
You can find inverted qrcode on google or just invert image in bitmap editor and try it :) e.g.

**Comment by EddyVerbruggen** [comment]
@MurilloBrand You're spot on. Let me release 9.1.2 today, with a rebuilt jar. Thanks!

**Pull request merged**